src/components/tools/geometry-tool/use-movable-line-dialog.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+import MovableLineSvg from "../../../clue/assets/icons/geometry/movable-line.svg";
+import { useCustomModal } from "../../../hooks/use-custom-modal";
+import { getBoundingBoxIntersections, solveForY } from "../../../models/tools/geometry/jxg-movable-line";
+import { round } from "lodash";
```

Comment:
We generally try to order imports with `node_modules` imports before local imports, so this should be moved ⬆️  at or near the top of the imports.